helm/prometheus-rules/templates/alerting-rules/aws.workload-cluster.rules.yml

```diff
@@ -60,7 +60,7 @@ spec:
       annotations:
         description: '{{`Control plane node in HA setup is down for a long time.`}}'
         opsrecipe: master-node-missing/
-      expr: sum(kubernetes_build_info{app="kubelet"} unless on (node) kube_node_role{role=~"control-plane|master"}) by (cluster_id) == 2
+      expr: sum by (cluster_id) (kubernetes_build_info{app="kubelet"} * on (node) kube_node_role{role="control-plane"}) == 2 or sum by (cluster_id) (kubernetes_build_info{app="kubelet"} * on (node) kube_node_role{role="master"}) == 2
```

Comment:
suggestion:
```
expr: sum by (cluster_id) (kubernetes_build_info{app="kubelet"} * on (node) kube_node_role{role=~"control-plane|master"}) == 2
```

Reply:
Not possible because on 18+ clusters, kube_node_roles has 2 metrics for the same node because they have both control-plane and master roles set

Reply:
![image](https://user-images.githubusercontent.com/2787548/235916871-d7b0390c-8daa-427d-9f67-63650629d14b.png)